tests/phpunit.xml

```diff
@@ -11,28 +11,20 @@
     stderr="true"
     bootstrap="./phpunit.bootstrap.php"
 >
-    <php>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak_vendors" />
```

Comment:
I removed this so that we know about deprecations.

Reply:
I think we can keep this?

https://symfony.com/doc/current/components/phpunit_bridge.html#making-tests-fail

> By using the "weak_vendors" value, deprecations that are triggered outside the vendors directory will make the test suite fail, while deprecations triggered from a library inside it will not, giving you the best of both worlds.

This is want we want? :blush:

Reply:
makes sense, done!